index.html

```diff
@@ -75,16 +75,15 @@
                     <li class="nav-item">
                         <a class="nav-link  font-weight-bold nav-items" href="#footer">Contact</a>
                     </li>
-                    <li class="nav-item dropdown">
+
```

Comment:
Any particular reason to remove it?

Reply:
Add a link in the more section pointing to the booklist.html so that the user can navigate it from the homepage.